pkg/kosmosctl/rsmigrate/rsexport.go

```diff
@@ -0,0 +1,105 @@
+package rsmigrate
+
```

Comment:
[rsmigrate/rsexport.go](https://github.com/kosmos-io/kosmos/pull/101/files#diff-758b0c2da18948a96f480373079c32421c3d6c707eec0499c660422f06a03664)   you can rename to serviceexport for more readability

Reply:
modified